app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
```

Comment:
Probably no need to have all the generated devise views out already unless we plan to modify them. :thinking:

Reply:
I think they are probably going to be visually modified